indico/htdocs/sass/modules/event_display/_conferences.scss

```diff
@@ -232,3 +232,38 @@ ul.author-list {
         }
     }
 }
+
+.call-for-abstracts {
+
+    .dates {
+        margin-left: 30px;
+
+        td {
+            padding: 0 10px 10px 0;
+        }
+    }
+
+    .announcement {
+        color: $light-black;
+        padding: 10px 0;
+        margin-left: 30px;
+    }
+
+    .actions {
+        margin: 20px 0 0 30px;
+    }
+
+    .info-message-box {
+        display: flex;
```

Comment:
Just FYI, we also have the `.flex` class that you could put in the `.info-message-box` HTML element. That way you don't have to specify this rule.